### PR TITLE
Debianisation adjustment (from kuLa)

### PR DESCRIPTION
**Pull request by vorlock**
Adjusting execution directory ($scriptdir) to work +
after deploying as a *.deb

### CODE REVIEW

**Comment by andsens** [comment]
Isn't this very similar to what we're doing just 3 lines below your changes?

**Comment by vorlock** [comment]
It'a kind a very similar stuff, but line 15 makes a difference up there, so I would keep it as it is if somebody want to build Debian package out of it.
If you've got nicer solution for it just drop this patch and I'll be happy to pick your fix.

File: build-debian-cloud [hunk 1]
```diff
@@ -4,7 +4,13 @@
...
-scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+if [ -t "utils" ]; then
+  scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+else
+	scriptdir="/usr/share/build-debian-cloud-image"
+	cd $scriptdir
+fi
+
 if [ ! -d "$scriptdir/tasks" ]; then
 	scriptdir='/usr/share/build-debian-cloud'
```
Comment (vorlock) [inline build-debian-cloud:15]: My change allows this line to stay as it is which is going to be needed to run this stuff after installation from tarball (out of deb package)

**Comment by andsens** [comment]
Is this change still needed? I am still a little iffy about it since we change the directory. That shouldn't be necessary.

**Comment by vorlock** [comment]
This was/is for 'shell' version which to be honest I don't care too much atm and as it's deprecated (hopefully sooner then latter). I think you can close this PR.

BTW I have python package 'experimental ready' with even man pages (at least some of them). It's not uploaded yet as I need more testing for how deb behaves.

**Comment by andsens** [comment]
Alright. I was thinking about publishing the python version under a new name, since it is only marginally related to the previous one and does not only bootstrap for the cloud.

> BTW I have python package 'experimental ready' with even man pages (at least some of them). It's not uploaded yet as I need more testing for how deb behaves.

Sweeet. Can't wait to see it!
Documentation has for a while been quite far down on the list, but now that the framework has matured, I feel comfortable beginning to write it, without having to dread a rewrite with every design change.

**Pull request closed without merging**